addons/missileguidance/functions/fnc_onFired.sqf

```diff
@@ -40,16 +40,27 @@ TRACE_4("enabled",_shooter,_ammo,_projectile,typeOf _shooter);
 
 private _config = configFile >> "CfgAmmo" >> _ammo >> QUOTE(ADDON);
 
+private _configurationSource = _shooter;
```

Comment:
I will try to get a vanilla repro this evening, but I was testing with an Apache, where the hellfires are launched by the gunner.
For months until I investigated and made this change, my pilot guys had hellfires never track the laser if the laser wasn't the default code. I enabled tracing and found that the laser was being set on the player object rather than the vehicle.
https://github.com/acemod/ACE3/blob/629942885e111d99f054ec3e247b8454876b58b0/addons/laser/functions/fnc_keyLaserCodeChange.sqf#L62

Reply:
Does this work with https://github.com/acemod/ACE3/blob/e2ac18a05d5f646bc7cbc043bcc148fde4c0f5dd/addons/laser/functions/fnc_showVehicleHud.sqf#L102 ? I am worried about this breaking something somewhere else unintentionally, I think this was assumed to be the vehicle from the get-go